x-pack/plugins/screenshotting/server/formats/pdf/index.ts

```diff
@@ -100,12 +91,11 @@ export async function toPdf(
   { metrics, results }: CaptureResult
 ): Promise<PdfScreenshotResult> {
   let buffer: Buffer;
-  let pages: number;
   const shouldConvertPngsToPdf = layout.id !== 'print';
   if (shouldConvertPngsToPdf) {
     const timeRange = getTimeRange(results);
     try {
-      ({ buffer, pages } = await pngsToPdf({
+      ({ buffer } = await pngsToPdf({
```

Comment:
nit: is it worth making a distinction for the kind of screenshot export that's happening and still keeping the pages metrics we get when converting png to PDF?

Reply:
I'd prefer not to. If the result would be that the page count would be `1` for "preserve layout" and `undefined` for "print layout," there wouldn't be much value to that metric.